doc/getting-started.rst

```diff
@@ -17,6 +17,35 @@ All amounts are represented in the smallest unit (eg. cents), so USD 5.00 is wri
 
 See :doc:`features/parsing` for additional ways to instantiate a Money object from strings.
 
+Accepted integer values
+-----------------------
```

Comment:
Let's not worry too much on this. Getting started should help how to construct a Money object. I agree - it is the edge cases. But let's accept it now. If we want to move it to another section later, we can always do that.